website/docs/developer/development-guide/documentation/documentation-deployment.md

```diff
@@ -4,4 +4,73 @@ title: Deploying Documentation
 desc: Provides basic information on deploying ORCA documentation to nasa.github.io
 ---
 
-Information on deploying the documentation is TBD.
+The following sections provide information to ORCA users to create automated or
+manual releases on github. View the entire versioning and releases document 
+[here](https://github.com/nasa/cumulus-orca/blob/master/docs/release.md).
+
+## Automated Documentation Release on Github
+
+### Create a new ORCA release on github
+
+The release is automated in Bamboo, but the step must be manually started. If 
+you set the `RELEASE_FLAG` to `true` and the build steps passed, you will be 
+able to run the manual "Release" step in Bamboo.
+
+The CI release scripts will create a release based on the release version tag, 
+as well as uploading release artifacts to the Github release for the Terraform 
+modules provided by Cumulus. The Terraform release artifacts include:
+
+ * A multi-module Terraform .zip artifact containing filtered copies of the 
+   tf-modules, packages, and tasks directories for use as Terraform module sources.
+
+Just make sure to verify the appropriate .zip files are present on Github after 
+the release process is complete.
+
+### Merge the base branch into develop and master
+
+Merge the version update changes back into develop and master.
+
+If this is the latest version, you can simply create a PR to merge the release 
+branch into develop and master. Note: Do not squash this merge. Doing so will 
+make the "compare" view from step 4 show an incorrect diff, because the tag is 
+linked to a specific commit on the base branch.
+
+## Manual Documentation Release on Github
+
+Use the following steps to update Github documentation manually.
+
+::: Important
+
+* Users need **Node 1215** installed to perform manaul updates to documentation.
+* The `Deployment_Branch` must = **gh-pages**.
+:::
+
+```
```

Comment:
You may want to break this out into steps and lose some of the shell code here that is not needed. Things like `check_rc` are functions I created in the bash script.

I would probably start by saying clone the cumulus-orca repo. Maybe something like the following:

1. Clone the Cumulus Orca repo to your machine. Enter the repo directory and change to the proper branch.
   ```bash
   git clone https://github.com/nasa/cumulus-orca.git
   cd cumulus-orca
   git checkout release-X.Y.Z
   ```
2. Enter the `website` directory.
   ```bash
   cd website
   ```
3. Export the needed environment variables. Make sure to use your GitHub username and password.
   ```bash
   export DEPLOYMENT_BRANCH=gh-pages
   export GIT_USER="Your GIT Username"
   export GIT_PASS="Your GIT Password"
   ``` 
4. If your user email and user name configuration is not set for git, that must be done before running the deploy in order to avoid errors when pushing to the GitHub repository. If the config is set this step can be skipped.
   ```bash
   git config --global user.email "Your EMAIL Address"
   git config --global user.name "$GIT_USER"
   ```
5. Run the deployment script via npm.
   ```bash
   npm run deploy
   ```

Reply:
made updates as  suggested